src/Windowing/Silk.NET.Windowing.Desktop/GlfwWindow.cs

```diff
@@ -87,6 +86,9 @@ public GlfwWindow(WindowOptions options)
         /// <inheritdoc />
         public int RunningSlowTolerance { get; set; }
 
+        /// <summary>
+        /// Whether or not this window is queued to close.
+        /// </summary>
         public unsafe bool IsClosing => _glfw.WindowShouldClose(_windowPtr);

```

Comment:
Was there not any documentation to inherit for this method? If not, these doc comments should be moved to the relevant interface in Windowing.Common instead